state/data/__tests__/action.test.js

```diff
@@ -0,0 +1,38 @@
+import * as types from '../types';
```

Comment:
Please, rename this file according to testing file - `actions.test.js`

Reply:
Done.